### PR TITLE
Dedicated zookeeper support

### PR DESCRIPTION
**Pull request by h3nd24**
Added bundle option during cluster creation for the (soon to be released) Kafka feature of dedicated Zookeeper nodes

### CODE REVIEW

**Review by williamreynoldsinstaclustr — DISMISSED** [review]

File: examples/main.tf [hunk 1]
```diff
@@ -179,3 +179,28 @@ resource "instaclustr_kafka_user" "kafka_user_charlie" {
 data "instaclustr_kafka_user_list" "kafka_user_list" { 
   cluster_id = "${instaclustr_clustr.example_kafka.cluster_id}"
 }  
+
+resource "instaclustr_cluster" "example_kafka_dedicated_zookeeper" {
```
[the remark this answers is not in the document]
Comment (h3nd24) [reply examples/main.tf:183]: Yep I can, I just thought initially to create a separate one for clarity. Will do that.

File: instaclustr/resource_cluster.go [hunk 2]
```diff
@@ -359,7 +371,18 @@ func resourceClusterRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("cluster_id", cluster.ID)
 	d.Set("cluster_name", cluster.ClusterName)
 
-	nodeSize := cluster.DataCentres[0].Nodes[0].Size
+	nodeSize := ""
```
[the remark this answers is not in the document]
Comment (h3nd24) [reply instaclustr/resource_cluster.go:374]: Yeah that is the case currently. That's why we need to filter that out.

File: test/resource_cluster_test.go [hunk 3]
```diff
@@ -342,3 +368,32 @@ func testCheckClusterResize(hostname, username, apiKey, expectedNodeSize string)
 		return nil
 	}
 }
+
+func testKafkaDedicatedZookeeper(resourceName, hostname, username, apiKey, kafkaNodeSize, zookeeperNodeSize string) resource.TestCheckFunc {
```
[the remark this answers is not in the document]
Comment (h3nd24) [reply test/resource_cluster_test.go:372]: That maybe a good idea to combine the cluster for test user resource to be the same as the dedicated zookeeper one. Will do that. For the programatic commenting, I just thought it is more compact to have it in one .tf file. Obviously I can expand it, but then there will be several files with really similar contents. Happy for discussion.

File: instaclustr/resource_cluster.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (arjunrajshekhar) [reply instaclustr/resource_cluster.go:374]: Understood, thanks for clarifying!

File: instaclustr/resource_cluster.go [hunk 4]
```diff
@@ -251,9 +251,21 @@ func resourceCluster() *schema.Resource {
 										Optional: true,
 									},
 									"truststore": {
+										Type:     schema.TypeBool,
```
[the remark this answers is not in the document]
Comment (h3nd24) [reply instaclustr/resource_cluster.go:254]: not even sure how did that happen, thanks for notifying

**Review by arjunrajshekhar — CHANGES_REQUESTED** [review]
Looks good, thanks! Could the build version on the makefile please be bumped to v1.4.0? We should be good to go after that!

**Review by arjunrajshekhar — APPROVED** [review]

**Review by peter-toms — APPROVED** [review]

**Pull request merged**